src/figuresClasses.ts

```diff
@@ -1,11 +1,97 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
```

Comment:
It's no a string, it should be `triangle, circle or rectangle`
![image](https://github.com/user-attachments/assets/37c4e053-8158-4da6-9fa0-5f6b45f161cc)